src/index.js

```diff
@@ -1,5 +1,16 @@
+import {
+  CREATOR_ROLE_ID,
+  COORDINATOR_ROLE_ID,
+  MEMBER_ROLE_ID,
+} from './roles.js'
 export { plugin as MapeoStaticMapsFastifyPlugin } from './fastify-plugins/maps/static-maps.js'
 export { plugin as MapeoOfflineFallbackMapFastifyPlugin } from './fastify-plugins/maps/offline-fallback-map.js'
 export { plugin as MapeoMapsFastifyPlugin } from './fastify-plugins/maps/index.js'
 export { FastifyController } from './fastify-controller.js'
 export { MapeoManager } from './mapeo-manager.js'
+
+export const roles = /** @type {const} */ ({
```

Comment:
Curious about feedback on this API.

Reply:
not really an improvement, but alternative idea is to use `ROLE_ID` as a namespace and then have more semantic enum values that point to the internal constants e.g.

```ts
export const ROLE_ID = /** @type {const} */ ({
  CREATOR: CREATOR_ROLE_ID,
  COORDINATOR: ...,
  MEMBER: ...,
})
```

Reply:
I think I'd like the exports to stay as `CREATOR_ROLE_ID`, `COORDINATOR_ROLE_ID`, and `MEMBER_ROLE_ID`. I think it's a little clearer if you destructure them, especially because we might expose things like `CREATOR` in the future.

If you disagree I'm happy to make a followup change.